src/types/CType.ts

```diff
@@ -7,9 +7,9 @@
 import IPublicIdentity from './PublicIdentity'
 
 export interface ICTypeSchema {
-  $id: any
-  $schema: any
-  properties: any
+  $id: string
+  $schema: string
+  properties: object // TO DO: need to refine what properties are
```

Comment:
I believe this may have been resolved from moving the CTypeUtils into the demo client.